src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -2203,6 +2207,48 @@ namespace AirflowNetwork {
                 ShowContinueError(m_state, "..Default value \"SkylineLU\" will be used.");
             }
 
+            // Get inputs for duct sizing
+            AirflowNetworkSimu.AFNDuctAutoSize = false;
+            if (NumAlphas == 10) {
+                if (!UtilityRoutines::SameString(Alphas(10), "None")) {
```

Comment:
Use `getEnumerationValue` pattern for this. It already exists in 100+ places in the code.

Reply:
@amirroth You are right. There are more than 100. Therefore, I am going to keep as is and discuss with @jasondegraw hoe to make changes totally.